sdk/python/tests/test_feature_set.py

```diff
@@ -11,21 +11,24 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Shouldn't we also add the other missing tests like for your update methods?

Reply:
Ok will add it